packages/remix-vercel/package.json

```
@@ -16,13 +16,10 @@
   },
   "devDependencies": {
     "@types/supertest": "^2.0.10",
-    "@vercel/node": "^1.8.3",
+    "@vercel/node": "^2.0.1",
     "node-mocks-http": "^1.10.1",
     "supertest": "^6.1.5"
   },
-  "peerDependencies": {
-    "@vercel/node": "^1.8.3"
-  },
```

We should keep `@vercel/node` in `peerDependencies`.
We should however support having v2.

```suggestion
  "peerDependencies": {
    "@vercel/node": "^1.8.3 || ^2.0.0"
  },
```

---

Its only use to import a type. Why is it needed for `peerDependencies`?

---

@styfle The imported types are used in both `GetLoadContextFunction` & `RequestHandler`, which are both exported by the adapter, so it should at least be in `peerDependencies`